css/styles.css

```diff
@@ -201,20 +217,21 @@
 
 .ab-column--submit-service {
     width: 342px;
-    margin-top: 56px;
+    margin-top: 20px;
+    padding-right:20px;
+	
 }
 .ab-column--submit-service p {
     padding: 5px 0;
     margin: 0;
     text-align: center;
     width: 100%;
-    background:#f6f6f6;
```

Comment:
I actually tried to discuss that on slack the other day, but you insisted on talking about privacy hints instead. While the old (and here restored) version with top and bottom border were perfectly fine, the slightly changed background-color, while being a good idea in the beginning, did not provide any additional value to the users and instead - and due low contrast looked a bit broken and not inline with the rest of ASBs GUI.